src/main/java/com/thalesgroup/hudson/plugins/cccc/CccccParser.java

```diff
@@ -223,4 +224,8 @@ public void setResultFilePath(FilePath resultFilePath) {
         this.resultFilePath = resultFilePath;
     }
 
+    @Override
+    public void checkRoles(RoleChecker roleChecker) throws SecurityException {
```

Comment:
Extend `MasterToSlaveFileCallable` instead of implementing your own role checker.
https://wiki.jenkins.io/display/JENKINS/Slave+To+Master+Access+Control#SlaveToMasterAccessControl-I'maplugindeveloper.WhatshouldIdo?

Reply:
Oh, this is part of the legacy code I didn't change. But I can modify that too.